lib/driver.js

```diff
@@ -226,11 +226,15 @@ class XCUITestDriver extends BaseDriver {
       }
     }
 
-    // make sure that the xcode we are using can handle the platform
-    if (parseFloat(this.opts.platformVersion) > parseFloat(this.iosSdkVersion)) {
-      let msg = `Xcode ${this.xcodeVersion.versionString} has a maximum SDK version of ${this.iosSdkVersion}. ` +
-                `It does not support iOS version ${this.opts.platformVersion}`;
-      log.errorAndThrow(msg);
+    if (typeof this.opts.webDriverAgentUrl === 'undefined' || this.opts.webDriverAgentUrl === null) {
```

Comment:
Thanks for the constructive feedback. I see your point. I think I will definitely go for the first option.